enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -617,6 +622,35 @@ def get_assignments_serializer(self, obj):
             context=context,
         )
         return serializer.data
+    
+    @extend_schema_field(LearnerCreditRequestSerializer(many=True))
+    def get_learner_requests(self, obj):
+        """
+        Return serialized learner credit requests associated with the user and policy's enterprise customer.
+        
+        Returns:
+            list: Serialized learner credit requests if policy has BNR enabled and user is authenticated.
+                  Empty list otherwise.
+        """
+        # Early returns if BNR not enabled or no user ID
+        if not obj.bnr_enabled:
+            return []
+            
+        lms_user_id = self.context.get('lms_user_id')
+        if not lms_user_id:
+            return []
+
+        # Get active learner credit requests for this user and enterprise customer        
+        learner_requests = LearnerCreditRequest.objects.filter(
+            user__lms_user_id=lms_user_id,
+            enterprise_customer_uuid=obj.enterprise_customer_uuid,
+            state__in=[SubsidyRequestStates.APPROVED, SubsidyRequestStates.REQUESTED],
+        )
```

Comment:
Shouldn't we further filter these records by policy_uuid (i.e., obj.uuid)? Right now, the query seems to return all requests tied to multiple policies and associates them with each policy object in the response.

Reply:
Based on our initial discussion, we assumed that the MVP would support only one B&R subsidy at a time. That’s why I didn’t include any additional subsidy checks here.

Reply:
I have added the policy filter as well.